source/_posts/2023-11-01-php-core-roundup-18.md

```diff
@@ -0,0 +1,145 @@
+
+Currently `round()`, `ceil()` and `floor()` functions return float numbers, but when using integers above `2^53` you get unexpected results due to loss of precision.
+
+RFC proposes to perform rounding for a given integer and returning the resulting integer if possible.
+
+### Under discussion: [Unbundle ext/imap, ext/pspell, ext/oci8, and ext/PDO_OCI](https://wiki.php.net/rfc/unbundle_imap_pspell_oci8) by Derick Rethans 💜
+
+RFC proposes to unbundle these extensions: remove them from the PHP source distribution, and move them to PECL.
+
+### Under discussion: [Multibyte for trim function mb_trim, mb_ltrim and mb_rtrim](https://wiki.php.net/rfc/mb_trim) by Yuya Hamada
+
+RFC proposes to add multibyte support for trim functions.
+
+### Under discussion: [Change the edge case of round()](https://wiki.php.net/rfc/change_the_edge_case_of_round) by Saki Takamachi
+
+RFC proposes to change the `round()` behavior, and stop expecting decimal behavior to float point and start expecting floating point to behave as floating point.
+
+
+<br>
+
+## Documentation
+
+While PHP 8.3 is just around the corner, the documentation available on [php.net](https://php.net), requires updating.
+
+George P. Banyard 💜 is tracking the progress for PHP 8.3 related changes in [php/doc-en#2796](https://github.com/php/doc-en/issues/2796), and also triaged issues in the docs and marked several of them as "good first time", which are ideal easy picks if you would like to start contributing to PHP docs. You can find the full list on [GitHub](https://github.com/php/doc-en/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+
+<br>
+
+## Merged PRs and Commits
+
+Following are some changes that did not go through an RFC process because they are either planned, bug fixes, or progressive enhancements.
+ 
+---
+
+### Full list of commits  since [PHP Core Roundup #17](/blog/2023/10/01/php-core-roundup-17/)
+
+Commits are in the order they were added, grouped by author in alphabetical order.
+
+<details markdown="1">
+  <summary>Click here to expand</summary>
+
+
+
+</details>
+<br>
+We are incredibly grateful for the commitment and dedication of all contributors. Stay tuned for next month's roundup as we continue to make PHP better together.
+
+<br>
+
+---
+
+## Support PHP Foundation
+
+At PHP Foundation, we support, promote, and advance the PHP language. We financially support six part-time PHP core developers to contribute to the PHP project. You can help support PHP Foundation at [OpenCollective](https://opencollective.com/phpfoundation) or via [GitHub Sponsors](https://github.com/sponsors/ThePHPF).
+
+A big thanks to all our sponsors — PHP Foundation is all of us!
+
+Follow us on Twitter [@ThePHPF](https://twitter.com/thephpf) to get the latest updates from the Foundation.
+
+💜️ 🐘
+
+> PHP Roundup is prepared by Ayesh Karunaratne from **[PHP.Watch](https://php.watch)**, a source for PHP News, Articles, Upcoming Changes, and more. 
```

Comment:
tbh, this feels a bit undeserving given it's @saundefined who carries the biggest weight. I think it's time we remove this line :)